features/admin/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+
+reviewers:
+- chengzhang1016
+- pruan-rht
+- zhouying7780
```

Comment:
Suggest remove yinzhou under admin, remove me from cli (keep yinzhou) and add me under admin

Reply:
@xingxingxia Updated.